src/main/java/seedu/exercise/model/ModelManager.java

```diff
@@ -283,11 +285,13 @@ public void setConflict(Conflict conflict) {
 
     /**
      * Returns an unmodifiable view of the list of {@code Exercise} backed by the internal list of
-     * {@code exerciseBook}.
+     * {@code exerciseBook} and sorted by Date.
      */
     @Override
     public ObservableList<Exercise> getFilteredExerciseList() {
-        return filteredExercises;
+        SortedList<Exercise> sortedExercises = new SortedList<>(filteredExercises, (o1, o2)
+            -> -o1.getDate().value.compareTo(o2.getDate().value));
```

Comment:
Why is there a `-` in front of the o1.getDate()?

Reply:
The date for Exercise list is arranged in descending order instead of ascending order. Makes more sense to have most recently completed exercises at the top of the list while exercises done months back get flushed downwards

Reply:
Alright. sounds good